pkgs/top-level/all-packages.nix

```diff
@@ -7786,6 +7786,8 @@ in
 
   targetcli = callPackage ../os-specific/linux/targetcli { };
 
+  targetisns = callPackage ../os-specific/linux/targetisns { };
```

Comment:
The attribute and folder name should probably match the `pname`.

Reply:
`openiscsi` and `openisns` are then inconsistent in the same way: the package name is `open-iscsi` but directory and attribute are called `openiscsi`. I changed everything to `target-isns` to be consistent.